pkgs/servers/tracing/tempo/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  version = "0.2.0";
+  pname = "tempo";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "grafana";
+    repo = "tempo";
```

Comment:
```suggestion
    owner = "grafana";
    repo = "tempo";
    rev = "v${version}";
```

Reply:
I don't see why sorting the attributes alphabetically would be necessary and I am not aware of any guideline in place covering this %)